src/cloud-api-adaptor/test/e2e/main_test.go

```diff
@@ -123,6 +123,10 @@ func TestMain(m *testing.M) {
 
 		// Get properties
 		props := provisioner.GetProperties(ctx, cfg)
+		if props["SECURE_COMMS"] == "true" {
+			testCase_secureComms_isActive = true
+			log.Info("Do setup secureComms is active")
+		}
```

Comment:
This code would not be needed if you followed the env var approach

Reply:
See above, I was unable to get this to work with an env var approach.